main.go

```diff
@@ -120,7 +120,10 @@ func main() {
 			"Sets the log output. Valid outputs are stdout and stderr").
 			Default("stdout").String()
 		awsRegion = kingpin.Flag("aws.region",
-			"Region for AWS elasticsearch").
```

Comment:
I don't think that this text needs to change. The region is still for the AWS cluster.